src/server.js

```diff
@@ -491,11 +441,16 @@ function performAuthentication(data) {
 	}
 
 	// Perform password checking
-	if (!Helper.config.public && Helper.config.ldap.enable) {
-		ldapAuth(client, data.user, data.password, authCallback);
-	} else {
-		localAuth(client, data.user, data.password, authCallback);
+	let auth = () => {
+		log.error("None of the auth plugins is enabled");
+	};
+	for (let i = 0 ; i < authPlugins.length ; ++i) {
```

Comment:
Can you remove the spaces before semilicons please.

Reply:
You should add `semi-spacing: [2, {"before": false, "after": true}]` to the `.eslintrc` then. ;) ([doc](https://eslint.org/docs/rules/semi-spacing))